### ISSUE TITLE
Integrate Sound Cloud

### ISSUE THREAD

**Issue by shnibble** [shnibble/OvhaiWebsite#30]
We need all of the tracks uploaded to Sound Cloud and set to public in order to stream them through the site. "Steaming" them will take a few seconds to load the track on our site but makes it very light on our end (i.e. Sound Cloud does all the work). This also lets us use our own buttons and UI however we'd like.

Once all tracks have been added to Sound Cloud please reassign to me and I'll integrate to the site.

**Comment by Meopicad** [comment]
Ok so we can keep the design that you have and just have the streaming done through soudcloud?

**Comment by shnibble** [comment]
Correct. It would look exactly like it does now just music would actually play and the $ button would redirect you to a Sound Cloud purchase area or whatever method we use for transactions. You can of course redesigned the function buttons for the tracks if you want. Those are just something I whipped up in paint.net

**Comment by Meopicad** [comment]
The goal is to be able to purchase directly from the website, so the purchase button will add the track to a shopping cart. There should also be a purchase button for the entire album. 

Additional merchandise should be displayed on the Albums page as well, we will have t-shirts stickers etc.
